tests/resources/user_fields-D8.php

```diff
@@ -30,7 +27,17 @@
   'field_user_text_long' => 'Super duper private info',
   'field_user_text_with_summary' => 'Private',
 ];
-$user = User::create($values);
+
+$user = User::create([
+  'name' => $args[2],
+  'mail' => $args[3],
+  'pass' => 'password',
+]);
+
+foreach ($values as $field_name => $value) {
+  $user->set($field_name, $value);
```

Comment:
Using this method creates a more obvious failure when the values are not set, which is nice.